model/task/db.go

```diff
@@ -1180,25 +1175,67 @@ func FindOld(query db.Q) ([]Task, error) {
 	return tasks, err
 }
 
-// FindOldWithDisplayTasks finds display and execution tasks in the old collection
-func FindOldWithDisplayTasks(query db.Q) ([]Task, error) {
+// FindOld returns all task from the old tasks collection that satisfies the query.
```

Comment:
Can you also mention that on top of finding the task, it also merges test results?